powerauth-webauth/src/main/java/io/getlime/security/powerauth/app/webauth/configuration/PowerAuthWebServiceConfiguration.java

```diff
@@ -60,14 +70,55 @@ public PowerAuthServiceClient powerAuthClient(Jaxb2Marshaller marshaller) {
             ClientInterceptor interceptor = securityInterceptor();
             client.setInterceptors(new ClientInterceptor[] { interceptor });
         }
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificate) {
+            trustAllCertificates();
+        }
         return client;
     }
 
     @Bean
     public PushServerClient pushServerClient() {
         PushServerClient client = new PushServerClient();
         client.setServiceBaseUrl(powerAuthPushServiceUrl);
+        // whether invalid SSL certificates should be accepted
+        if (acceptInvalidSslCertificate) {
+            trustAllCertificates();
+        }
         return client;
     }
 
+    /**
+     * Activate trust in all SSL certificates including invalid ones for non-production use.
+     */
+    private void trustAllCertificates() {
```

Comment:
Ah, I see now, you probably mean duplicate setting in powerauth-webauth and powerauth-webauth-authentication projects. Yes, that should be fixed.